tests/Bridge/Doctrine/Orm/SubresourceDataProviderTest.php

```diff
@@ -267,6 +269,66 @@ public function testGetSubSubresourceItem()
+
+        // First manager (RelatedOwningDummy)
+        // Generated QueryBuilder
+        $qb = $this->prophesize(QueryBuilder::class);
+        $qb->select('IDENTITY(owningDummy_a1.owningDummy)')->shouldBeCalled()->willReturn($qb);
+        $qb->from(RelatedOwningDummy::class, 'owningDummy_a1')->shouldBeCalled()->willReturn($qb);
+        $qb->andWhere('owningDummy_a1.id = :id_p1')->shouldBeCalled()->willReturn($qb);
+
+        // RelatedOwningDummy classMetaData
+        $classMetadataProphecy = $this->prophesize(ClassMetadata::class);
+        $classMetadataProphecy->hasAssociation('ownedDummy')->willReturn(true)->shouldBeCalled();
+        $classMetadataProphecy->getAssociationMapping('ownedDummy')->shouldBeCalled()->willReturn(['type' => ClassMetadata::ONE_TO_ONE]);
+        $classMetadataProphecy->getTypeOfField('id')->willReturn(DBALType::INTEGER)->shouldBeCalled();
+
+        // RelatedOwningDummy Manager
+        $relatedOwningDummyManagerProphecy = $this->prophesize(EntityManager::class);
+        $relatedOwningDummyManagerProphecy->createQueryBuilder()->shouldBeCalled()->willReturn($qb->reveal());
+        $relatedOwningDummyManagerProphecy->getClassMetadata(RelatedOwningDummy::class)->shouldBeCalled()->willReturn($classMetadataProphecy->reveal());
+
+        $managerRegistryProphecy->getManagerForClass(RelatedOwningDummy::class)->shouldBeCalled()->willReturn($relatedOwningDummyManagerProphecy->reveal());
+
+        // Result Query
+        $result = new \stdClass();
+        $queryProphecy = $this->prophesize(AbstractQuery::class);
+        $queryProphecy->getOneOrNullResult()->shouldBeCalled()->willReturn($result);
+
+        // Second manager (RelatedOwnedDummy)
+        // RelatedOwnedDummy classMetaData
+        $classMetadataOwnedProphecy = $this->prophesize(ClassMetadata::class);
+        $classMetadataOwnedProphecy->hasAssociation('owningDummy')->willReturn(true)->shouldBeCalled();
+        $classMetadataOwnedProphecy->getAssociationMapping('owningDummy')->shouldBeCalled()->willReturn(['type' => ClassMetadata::ONE_TO_ONE]);
+        $classMetadataOwnedProphecy->getTypeOfField('id')->willReturn(DBALType::INTEGER)->shouldBeCalled();
+
+        $queryBuilder = $this->prophesize(QueryBuilder::class);
+        $queryBuilder->getQuery()->shouldBeCalled()->willReturn($queryProphecy->reveal());
+        $queryBuilder->setParameter('id_p1', 1, DBALType::INTEGER)->shouldBeCalled()->willReturn($queryBuilder);
+
+        $repositoryProphecy = $this->prophesize(EntityRepository::class);
+        $repositoryProphecy->createQueryBuilder('o')->shouldBeCalled()->willReturn($queryBuilder->reveal());
+
+        $ownedDummyManagerProphecy = $this->prophesize(EntityManager::class);
+        $ownedDummyManagerProphecy->getClassMetaData(RelatedOwnedDummy::class)->willReturn($classMetadataOwnedProphecy->reveal());
+        $ownedDummyManagerProphecy->getRepository(RelatedOwnedDummy::class)->willReturn($repositoryProphecy->reveal());
```

Comment:
@soyuka here I tried to remove shouldBeCalled but I still have issue with prophecy like I said in my comment... But in the code We actually called both getClassMetaData and getRepository.

Reply:
I'm checking but I'm giving a class so may take a bit :p

Reply:
I think I found something... With the behat test failed, I see that in fact the RelatedOwnedDummy and RelatedOwningDummy are not in relation together but both with Dummy (weird isn't it :nerd_face:):
`targetEntity="Dummy"`